app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -1204,6 +1209,66 @@ fun findAndUpdateComment(
     }
 }
 
+fun findAndUpdatePostCreator(
+    posts: List<PostView>,
+    person: Person,
+): List<PostView> {
+    val newPosts = posts.toMutableList()
+    newPosts.replaceAll {
```

Comment:
I couldn't find a better way to alter several items for an immutable list. 

I would've liked to use `.filter { }.forEach { }`, but it didn't work since the internal items are immutable.